src/lib/Tab/LocationView/TranslationsTab.php

```diff
@@ -132,4 +158,27 @@ private function createTranslationDeleteForm(Location $location, array $language
 
         return $this->formFactory->deleteTranslation($translationDeleteData);
     }
+
+    /**
+     * @param $limitations
+     *
+     * @return string[]
+     */
+    private function getLimitationLanguageCodes($limitations): array
```

Comment:
Does anyone have a nice idea where we could extract both this method?

Reply:
PermissionUtil class that @mikadamczyk is adding in his PR?